R/messages.R

```diff
@@ -1,4 +1,45 @@
+#' Functions to concat strings to messages used e.g. in errors
+#' @export
 messages <- list(
+  errorGetEntityMultipleOutputs = function(path, container, optionalMessage = NULL) {
+    callingFunction <- .getCallingFunctionName()
+    paste0(
+      callingFunction, ": the path '", toString(path), "' located under container '",
+      container$path,
+      "' leads to more than one entity! Use 'getAllXXXMatching'",
+      "to get the list of all entities matching the path, where XXX stands for the entity type", optionalMessage
+    )
+  },
+  errorEntityNotFound = function(path, container, optionalMessage = NULL) {
+    callingFunction <- .getCallingFunctionName()
+    paste0(
+      callingFunction, ": No entity exists for path '", toString(path), "' located under container '",
+      container$path,
+      "'!", optionalMessage
+    )
+  },
+  errorResultNotFound = function(path, individualId, optionalMessage = NULL) {
+    callingFunction <- .getCallingFunctionName()
+    paste0(
+      callingFunction, ": No results exists for path '", toString(path), "' for individual IDs ",
```

Comment:
There is no function called `toString()` in the utilities package. You will also need to move it to this package.

Reply:
Eeeh it's base :D